perllib/FixMyStreet/App/Controller/Admin.pm

```diff
@@ -1163,7 +1163,7 @@ sub fetch_all_bodies : Private {
 sub fetch_body_areas : Private {
     my ($self, $c, $body ) = @_;
 
-    my $children = $body->first_area_children;
+    my $children = $c->cobrand->call_hook('admin_fetch_inspector_areas', $body) || $body->first_area_children;
```

Comment:
After all the fixups, I think what's fallen out is this hook can be removed? Because it now doesn't do anything, as fetch_area_children does it all anyway.